### PR TITLE
feat: svelte 5 POC

### PR DESCRIPTION
**Pull request by PuruVJ**
Current REPL's state is a mess. Bunch of `get(store)` in .js files. This simplifies all that to use $state

### CODE REVIEW

File: packages/repl/src/lib/Bundler.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ export default class Bundler {
 		const hash = `${packages_url}:${svelte_url}`;
 
 		if (!workers.has(hash)) {
-			const worker = new Worker();
+			const worker = new Worker(bundler_worker, { type: 'module' });
```
Comment (paoloricciuti) [inline packages/repl/src/lib/Bundler.js:16]: I think module workers still doesn't work on Firefox

Comment (PuruVJ) [reply packages/repl/src/lib/Bundler.js:16]: https://caniuse.com/?search=worker%20module

**Comment by benmccann** [comment]
It'd probably be easier to review if the prettier upgrade were a separate PR

File: packages/repl/src/lib/Bundler.js (hunk 1 above)
Comment (paoloricciuti) [reply packages/repl/src/lib/Bundler.js:16]: The first result is only for `Worker` if you scroll to the last result `ES Modules in workers` you can see that Firefox doesn't have support yet

**Comment by PuruVJ** [comment]
It has to be done here, otherwise using snippets break prettier across entire project.

File: packages/repl/src/lib/Bundler.js (hunk 1 above)
Comment (PuruVJ) [reply packages/repl/src/lib/Bundler.js:16]: That's for service worker

**Comment by benmccann** [comment]
Couldn't you upgrade prettier first and then do the Svelte 5 upgrade after that?

**Comment by benmccann** [comment]
thank you!

closing this since we have the omnisite, but happy to get any of these changes there

**Pull request closed without merging**